javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,11 @@
+/* exported oddOrEven */
+function oddOrEven(numbers) {
+  var newArray = [];
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      numbers[i] = 'even';
```

Comment:
Again, you are changing the original array permanently here, which is another example of a "side effect". Unless it is the intended purpose of the function, a function should never make changes to the content of any arrays or objects given as arguments to the function.